rpmlb/__init__.py

```diff
@@ -1,25 +1 @@
-"""Top-level module
-"""
-
-import logging
-
 __version__ = '1.0.0'
```

Comment:
I found below logic to manage version number. Does it look better than our current code?
https://github.com/rebase-helper/rebase-helper/blob/master/setup.py#L24